packetbeat/statsdns/statistics_dns.go

```diff
@@ -101,62 +101,80 @@ var (
 	IpsServer                    []string
 	UrlAnnouncementDeployFromBam string
 	MapViewIPs                   map[int]map[string][]string
+	QStatDNS                     *QueueStatDNS
```

Comment:
There are performance issues (CPU) in FindClientInView -> regex.FindString that I have told c.AnhQ verbally, please help to check.